pkg/log/log.go

```diff
@@ -102,15 +136,25 @@ func SessionLogger(log Logger) mux.MiddlewareFunc {
 				path = path + "?" + raw
 			}
 
-			l.Info().
-				Str("clientIP", clientIP).
+			log.Str("clientIP", clientIP).
 				Str("method", method).
 				Str("path", path).
 				Int("statusCode", statusCode).
 				Str("latency", latency.String()).
 				Int("bodySize", bodySize).
 				Interface("headers", headers).
 				Msg("HTTP API")
+
+			if (method == http.MethodPost || method == http.MethodPut ||
+				method == http.MethodPatch || method == http.MethodDelete) &&
+				(statusCode == http.StatusOK || statusCode == http.StatusCreated || statusCode == http.StatusAccepted) {
+				audit.Info().
+					Str("subject", username).
```

Comment:
Another issue is that we may not have a "username" if Bearer AuthN is used. So maybe we should also add ClientIP (above) into this record.